contracts/Template.sol

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Don't have to do this now, but we could update this once the template is updated to use our new template structure (https://github.com/aragon/aragon-react-boilerplate/pull/69 should be done soon)